test/integration/test-one.sh

```diff
@@ -83,8 +83,8 @@ cp -R "${TESTDIR}"/* "${RUNDIR}/"
 run-step() {
     local script="$1"
     if [ ! -x "$script" ]; then
-        log-warn "skipping \"$script\"; not executable"
-        return
+        log-warn "Failing: \"$script\"; not executable"
+        return 1
```

Comment:
We have the `fail-now` function that can be used to fail the test.

Reply:
ah I had looked at that and got mixed up on behavior of `exit` within a function :)

will fix